README.md

```diff
@@ -381,47 +396,61 @@ Note: You need to match the above format exactly.
 
 ### Manual provisioning
 
-If you want to provision a sandbox outside of a GitHub pull request, you can do
+If you want to create an instance outside of a GitHub pull request, you can do
 so from the shell:
 
 ```python
-from instance.models.instance import SingleVMOpenEdXInstance
-from instance.tasks import provision_instance
-
-instance = SingleVMOpenEdXInstance.objects.create(
-    sub_domain='dogwood.sandbox',
-    name='Dogwood',
-    fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+from instance.models.openedx_instance import OpenEdXInstance
+
+instance = OpenEdXInstance.objects.create(
+    name='Dogwood sandbox',
+    sub_domain='dogwood',
+    # The rest of the parameters are all optional:
+    email='myname@opencraft.com',
+    openedx_release='named-release/dogwood',
     configuration_version='named-release/dogwood',
-    forum_version='named-release/dogwood',
-    notifier_version='named-release/dogwood',
-    xqueue_version='named-release/dogwood',
-    certs_version='named-release/dogwood',
-    ansible_source_repo_url='https://github.com/edx/configuration.git',
+    configuration_source_repo_url='https://github.com/edx/configuration.git',
+    configuration_extra_settings='',
+    use_ephemeral_databases=False,
 )
 
-instance.ansible_extra_settings = """
-# Add custom ansible settings here, as yaml
+# Optionally, set custom ansible variables/overrides:
+instance.configuration_extra_settings = """
 NGINX_ENABLE_SSL: true
 """
-
 instance.save()
-provision_instance(instance.pk)
 ```
 
-To reprovision an instance from the shell, simply run the `provision_instance`
-task again:
+Once the instance is created, use the web UI to review the instance
+configuration, then use the "Launch new AppServer" button to provision a server.
+
+Once the server is ready, select it in the UI and click "Activate this app
+server". (You can also do this in advance, during provisioning, if you want the
+DNS updated sooner and aren't concerned about the DNS pointing to a potentially
+broken server, in the case the provisioning should fail.)
+
+
+To change an instance's parameters, if that instance is not controlled by a pull
+request:
+
+First, note the instance's ID (will be in the URL of that instance in
+the web UI, or get it in the shell as `instance.ref.id`). Then, load the
+instance, make changes, and save:
 
 ```python
-instance = SingleVMOpenEdXInstance.objects.get(name__contains='...')
-provision_instance(instance.pk)
+instance = InstanceReference.objects.get(id=20).instance
+# Update settings of instance:
+instance.edx_platform_commit = 'master'
+# Save:
+instance.save()
 ```
 
+
```

Comment:
@bradenmacdonald It might be worth mentioning here that a user should spawn a new app server after making changes to the instance (to make them take effect).